### PR TITLE
feat(reactions): input/submit components

### PR DESCRIPTION
**Pull request by Confidenceman02**
screen shot 2017-11-22 at 4 44 12 pm
added file structure to reactions, implemented submiReaction compoenent and styled

### CODE REVIEW

File: src/components/Input/index.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import style from './Input.scss'
+
+const Input = () => {
+  return (
+    <input type="text" placeholder="Enter comment..." className={style.input} />
```
Comment (Confidenceman02) [inline src/components/Input/index.js]: prop for placeholder

File: src/common/_variables.scss [hunk 2]
```diff
@@ -1,10 +1,17 @@
 // Colour Palette
 $alto: #F0F0F0;
+$alabaster: #FAFAFA;
```
Comment (rodleviton) [inline src/common/_variables.scss]: Love the old Alabaster!

File: src/components/Button/Button.scss [hunk 3]
```diff
@@ -25,16 +25,19 @@
...
   align-self: center;
   &:after {
-    content: "+";
-    color: #FFBD00;
-    margin-left: 11px;
-    font-size: 3rem;
-    display: inline-block;
-    vertical-align: middle;
-    margin-bottom: 7px;
+    @include btn-after-icon-style('+');
```
Comment (rodleviton) [inline src/components/Button/Button.scss]: Yup!

File: src/components/Button/Button.scss [hunk 4]
```diff
@@ -25,16 +25,19 @@
...
   }
 }
+
+.submit {
+  height: 40px;
+  padding: 6px 20px;
+  &:after {
+    @include btn-after-icon-style('>')
```
Comment (rodleviton) [inline src/components/Button/Button.scss]: I love the mixin but the actual icons should be generated from font-awesome/ionicons.

File: src/components/Button/index.js [hunk 5]
```diff
@@ -5,7 +5,8 @@ import classnames from 'classnames'
 
 // properties could have class combinations
 const btnClasses = {
-  addPixels: 'add-pixels'
+  addPixels: 'add',
```
Comment (rodleviton) [inline src/components/Button/index.js]: I think these could just be generic. No need for `addPixels`. Button type could just be `add` or `submit`

File: src/components/Header/index.js [hunk 6]
```diff
@@ -13,11 +13,13 @@ class Header extends Component {
     const { user } = this.props
 
     return (
-      <header className={styles.header}>
-        <Logo />
-        <AddPixels />
-        <UserLogin user={user} loginAction={authenticateUserWithGithub} />
-      </header>
+      <div>
```
Comment (rodleviton) [inline src/components/Header/index.js]: Extra div?

File: src/components/Input/Input.scss [hunk 7]
```diff
@@ -0,0 +1,18 @@
+@import '../../common/_variables.scss';
+
+.input {
```
Comment (rodleviton) [inline src/components/Input/Input.scss]: Very tidy.

**Comment by rodleviton** [comment]
Seriously dude. Great work!

**Comment by rodleviton** [comment]
Also happy for it to merged when you are.

**Pull request merged**